bin/cppman

```diff
@@ -161,7 +161,8 @@ def main():
     try:
         pid = cm.man(args[0])
     except RuntimeError as e:
-        print(e)
+        print(e, file=sys.stderr)
+        sys.exit(16)
```

Comment:
Can you make it `sys.exit(1)`? code 16 doesn't seem appropriate here.

Reply:
This is because [`man` command returns code `16`.](https://man7.org/linux/man-pages/man1/man.1.html#EXIT_STATUS:~:text=16%20%20%20%20%20At%20least%20one%20of%20the%20pages/files/keywords%20didn%27t%20exist%20or%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20wasn%27t%20matched%2e)
Don't you think it should use the same exit code?

Reply:
Thanks for the clarification, returning 16 sounds good then.